internal_cert_management/scripts/lego.sh

```
@@ -0,0 +1,64 @@
+        --read-only \
+        -it goacme/lego \
+        --email "$ADMIN_EMAIL" \
+        --key-type=$type \
+        --dns "$LEGO_DNS_PROVIDER" \
+        --dns.resolvers "$LEGO_DNS_RESOLVERS" \
+        --domains '*.'$subdomain --domains $subdomain \
+        renew
+    done
+done
+
+for type in rsa2048
```

Shellcheck (0.11.0)

[warning] This loop will only ever run once. Bad quoting or missing glob/expansion?

(SC2043)